packages/fast-components-class-name-contracts/src/image/index.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * The classname contract for the media component

```

Comment:
If we are naming this image, the comment should use image (as opposed to media)

Reply:
Thanks! I just changed the naming over yesterday and must have missed a few.